app/assets/stylesheets/pages/_place-form.scss

```diff
@@ -0,0 +1,16 @@
+.new-place,
+.edit-place {
+  padding: 0 0 $base-spacing 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 0 $base-spacing` instead of `0 0 $base-spacing 0`